packages/core/src/PlayerContextProvider.js

```diff
@@ -135,6 +141,7 @@ export class PlayerContextProvider extends Component {
       setVolumeInProgress: false,
       // initialize awaitingPlay from autoplay prop
       awaitingPlay: props.autoplay && isPlaylistValid(props.playlist),
+      awaitingLoad: false,
```

Comment:
This could be `true`,  and then set to `false` after calling `setMediaElementSources`, but it doesn't serve any purpose and it would cause a re render.

Reply:
Sounds good to me. Maybe it should be called `awaitingForceLoad` since that's basically what we're doing? What do you think